apps/src/sites/studio/pages/blockly.js

```diff
@@ -1,4 +1,17 @@
-import Blockly from '@code-dot-org/blockly';
+import experiments from '@cdo/apps/util/experiments';
+
+import CDOBlockly from '@code-dot-org/blockly';
```

Comment:
just making sure this comment from slack doesn't get lost... adding this line as-is does not look shippable, as it will be a huge increase in JS download size for any blockly level. I recommend creating a separate googleblockly.js entry point (a peer of this file) and conditionally including it here: https://github.com/code-dot-org/code-dot-org/blob/029bc84a8f2e7f1cd35c83f345aae9ac8ddb8973/dashboard/app/views/levels/_apps_dependencies.html.haml#L51-L52

Reply:
credit to Anjali for recognizing the problem... I'm just offering a possible solution :)

Reply:
I guess i put the comment on the wrong line here, and now the comment doesn't make any sense since the offending line has been removed. anyway, this looks good now!